agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -211,6 +244,38 @@ func getSessionWorkersLimit(ma apicontainer.ManagedAgent) int {
 	return limit
 }
 
+var GetExecAgentLogConfigFile = getAgentLogConfigFile
+
+func getAgentLogConfigFile() (string, error) {
+	hash := getExecAgentConfigHash(execAgentLogConfigTemplate)
+	logConfigFileName := fmt.Sprintf(logConfigFileNameTemplate, hash)
+	// check if config file exists already
+	logConfigFilePath := filepath.Join(ECSAgentExecConfigDir, logConfigFileName)
+	if fileExists(logConfigFilePath) && validLogConfigExists(logConfigFilePath, hash) {
+		return logConfigFileName, nil
+	}
+	// create new seelog config file
+	if err := createNewExecAgentConfigFile(execAgentLogConfigTemplate, logConfigFilePath); err != nil {
+		return "", err
+	}
+	return logConfigFileName, nil
+}
+
+func validLogConfigExists(configFilePath, expectedHash string) bool {
+	config, err := getFileContent(configFilePath)
+	if err != nil {
+		return false
+	}
+	hash := getExecAgentConfigHash(string(config))
+	return strings.Compare(expectedHash, hash) == 0
```

Comment:
nit: maybe we can use `==` here, or we might need `strings.Compare()` for case-insensitive comparision